charts/trino/values.yaml

```diff
@@ -948,6 +948,13 @@ serviceMonitor:
   # serviceMonitor.enabled -- Set to true to create resources for the
   # [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator).
   enabled: false
+  # -- Override apiVersion for the ServiceMonitor resource.
+  # @raw
+  # Example:
+  # ```yaml
+  # apiVersionOverride: azmonitoring.coreos.com/v1
+  # ```
+  apiVersionOverride:
```

Comment:
Maybe it could be `apiVersion` with `monitoring.coreos.com/v1` as the default? You wouldn't need the template then.

Reply:
Thanks! I will change it.